### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by bardobrado** [spyder-ide/spyder#6410]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\client.py", line 237, in stop_button_click_handler
    self.interrupt_kernel()
  File "C:\Anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\client.py", line 393, in interrupt_kernel
    self.shellwidget.request_interrupt_kernel()
  File "C:\Anaconda3\lib\site-packages\qtconsole\frontend_widget.py", line 345, in request_interrupt_kernel
    self.interrupt_kernel()
  File "C:\Anaconda3\lib\site-packages\qtconsole\frontend_widget.py", line 638, in interrupt_kernel
    self.kernel_manager.interrupt_kernel()
  File "C:\Anaconda3\lib\site-packages\jupyter_client\manager.py", line 408, in interrupt_kernel
    raise RuntimeError("Cannot interrupt kernel. No kernel is running!")
RuntimeError: Cannot interrupt kernel. No kernel is running!

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.6). Please update.

**Comment by CAM-Gerlach** [comment]
For the record, a duplicate of #5568 . You can update with ``conda update spyder`` from the Anaconda prompt.
